src/disclosures/js/dispatchers/get-api-values.js

```diff
@@ -55,7 +55,7 @@ var getApiValues = {
       // TODO: the user should be notified of errors
       error: function( req, status, err ) {
         // console.log( 'API: fetchSchoolData', status, err );
```

Comment:
Can we remove the commented out console.logs? Also, the TODOs for each section where the work has now been done? (Yay, work done!)